### PR TITLE
zh-translation: /zh/docs/reference/config/analysis/ist0132/index.md

### PR DESCRIPTION
**Pull request by MXuDong**
[ ] Configuration Infrastructure
[x] Docs
[ ] Installation
[ ] Networking
[ ] Performance and Scalability
[ ] Policies and Telemetry
[ ] Security
[ ] Test and Release
[ ] User Experience
[ ] Developer Infrastructure

### CODE REVIEW

**Review by SataQiu — COMMENTED** [review]
/ok-to-test

File: content/zh/docs/reference/config/analysis/ist0132/index.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
+---
+title: VirtualServiceHostNotFoundInGateway
+layout: analysis-message
+owner: istio/wg-user-experience-maintainers
+test: no
+---
+
+当一个虚拟服务声明了 `host`，但是无法找到相应的网关时，会出现此消息。
```
Comment (kebe7jun) [inline content/zh/docs/reference/config/analysis/ist0132/index.md:8]: vs

File: content/zh/docs/reference/config/analysis/ist0132/index.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+      protocol: HTTP
+{{< /text >}}
+
+您将会收到以下信息：
+
+{{< text plain >}}
+Warning [IST0132] (VirtualService testing-service.default testing.yaml:8) one or more host [wrong.com] defined in VirtualService default/testing-service not found in Gateway istio-system/testing-gateway.
+{{< /text >}}
+
+在这个示例中，虚拟服务 `testing-service` 拥有域名 `wrong.com` ，但是该域名没有声明在网关 `testing-gateway` 中。
```
Comment (kebe7jun) [inline content/zh/docs/reference/config/analysis/ist0132/index.md:60]: vs

File: content/zh/docs/reference/config/analysis/ist0132/index.md [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+
+## 解决方案{#how-to-resolve}
+
+确保虚拟服务中所有 `hosts` 都已绑定到了相应的网关 `hosts` 中。
```
Comment (kebe7jun) [inline content/zh/docs/reference/config/analysis/ist0132/index.md:64]: vs

File: content/zh/docs/reference/config/analysis/ist0132/index.md [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+  http:
+  - match:
+    - uri:
+        prefix: /
+    route:
+    - destination:
+        host: ratings
+{{< /text >}}
+
+同时还包含如下网关:
```
Comment (kebe7jun) [inline content/zh/docs/reference/config/analysis/ist0132/index.md:34]:
```suggestion
同时还包含如下 Gateway:
```

**Review by kebe7jun — APPROVED** [review]
LGTM

**Pull request merged**